enlighten/Controller.py

```diff
@@ -2477,6 +2496,20 @@ def open_log(self):
         # Interestingly a few threads explained that this will open the default text editor
         webbrowser.open(os.path.join(common.get_default_data_dir(), "enlighten.log"))
 
+    def graph_grid_toggle(self):
```

Comment:
Sorry, thought I'd clicked "approve" a couple days ago.

Reply:
No problem. I will add that. I was surprised you didn't have an opinion on the for/else clause. It's an oddity of python that I used since it seemed a rare but valid case so I had to have some fun with it. I'm planning to replace with something better before merging because imo it's not very intuitive and thus not readable.